dist/preview release/what's new.md

```diff
@@ -25,6 +25,7 @@
   - Added support for life time gradients. [Doc](https://doc.babylonjs.com/babylon101/particles#lifetime)
   - Added support for angular speed gradients. [Doc](https://doc.babylonjs.com/babylon101/particles#rotation)
 - Added SceneComponent to help decoupling Scene from its components ([sebavan](http://www.github.com/sebavan))
+- Added support AtlasTexture for GUI Image.

```

Comment:
please also add your GitHub alias to get full credits :)

Reply:
I forgot this :)